content/en/docs/collector/internal-telemetry.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Internal Telemetry
+weight: 25
+cSpell:ignore: journalctl kube otecol pprof tracez zpages
+---
+
+The Collector offers multiple ways to measure and monitor its own health. In
+this section, you'll learn how to enable internal observability.
```

Comment:
```suggestion
You can check the health of any Collector instance by checking its own
internal telemetry. Read on to learn how to use this data to diagnose and
troubleshoot Collector issues.
```

Suggesting this rewording to make it less about the Collector offers and more about what the user can do with this data.